src/tool.js

```diff
@@ -0,0 +1,33 @@
+exports.buildForm = tools => {
```

Comment:
"Tools" seems misleading that it should be an array of tools rather than it being an array of inputs.